substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jfr/events/NativeMemoryUsageTotalPeakEvent.java

```diff
@@ -28,14 +28,17 @@
 import jdk.jfr.Category;
 import jdk.jfr.Description;
 import jdk.jfr.Event;
+import jdk.jfr.Experimental;
 import jdk.jfr.Label;
 import jdk.jfr.Name;
 import jdk.jfr.StackTrace;
 
-@Name("svm.NativeMemoryUsageTotalPeak")
+/** Similar to the JFR event jdk.NativeMemoryUsageTotal, except that it tracks the peak usage. */
+@Experimental
+@Name("jdk.NativeMemoryUsageTotalPeak")
 @Label("Native Memory Usage Total Peak")
-@Description("Information about native memory peak usage of committed virtual memory and malloc.")
```

Comment:
I initially specified that the peak usage is reported with regard to committed virtual memory + mallocs, since we report both committed and reserved memory in the other NMT events. Do you think it's not necessary? I'm fine leaving out those details if it's already obvious.

Reply:
I think we should report both committed and reserved for the Native Image-specific events as well. I will change the code accordingly.